### DISCUSSION TITLE
Making sure that outside contributions are clearly identifiable - "documentation problem"

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#382] [Core Contributions]
Our documentation, how it is now isn't save:

Each time we update our documentation in autora each documentation gets "pulled" in from all the packages, we import via "!import ...". This means, that even if we approved the documentation A ones. The next time we want to add documentation B for a different project, documentation A will be "pulled" in again and might be altered since then.
This means that we would either have to look at all the documentation again and approve every package, we alter the way we pull in the documentation to create a static website (I played around a bit and that should be possible, though a bit tedious), or we clearly *mark* the outside documentation as such. (Or maybe somebody has a different solution?)

**Comment by younesStrittmatter** [comment]
@hollandjg: I played around with the mkdocs multirepo plugin, and it creates a "temp-folder" with all the pages for the packages that can be made permanent with the option multirepo: {clear: false} (as far as I understand). Maybe we can use that to create a static website once instead of importing each time.

**Reply by musslick** [reply]
I like the static solution as it allows us to "commit" to a given version of the package documentation, without fear of it being overwritten with something we don't approve to be part of autora.
